script/electron-builder-linux.yml

```diff
@@ -7,5 +7,14 @@ linux:
   target:
     - deb
     - rpm
+    - snap
     - AppImage
   maintainer: 'GitHub, Inc <opensource+desktop@github.com>'
+snap:
+  confinement: 'classic'
+  stagePackages:
+    - default
+    - git-core
```

Comment:
Can you clarify why this is needed? Ideally GitHub Desktop should be launching it's embedded Git package, and if something is not working as expected then I'd like to get that fixed in https://github.com/desktop/dugite-native

Reply:
I originally created the GitHub Desktop as a strictly confined Snap and that exposed a requirement for calling `git`. Sadly I don't have the logs for that anymore. If `git` is not required then `git-core` can be dropped from the `stage-packages:` and as the Snap is now using classic confinement it will find `git` if it is installed on the host system.

Reply:
I originally created the GitHub Desktop Snap with strict confinement, this highlighted an issue where `git` was being called (sorry I don't have the logs anymore) but due to the strict sandboxing wasn't permitted to execute the `git` binary installed on the host OS. Therefore I bundled `git-core` in the Snap.

Now that the GitHub Desktop Snap is using classic confinement, If there is a requirement to execute `git` it will be permitted to execute the `git` binary on the host OS. 

Would you prefer I drop `git-core` from the PR?

Reply:
Let me figure out how to test it locally and confirm `git-core` isn't required.